rustls/src/verify.rs

```diff
@@ -308,6 +366,36 @@ impl fmt::Debug for dyn ClientCertVerifier {
 }
 
 impl ServerCertVerifier for WebPkiVerifier {
+    fn verify(
+        &self,
+        input: ServerVerifyInput,
```

Comment:
Why would it make the semantics clearer (in Rust)? I think it's nice to be obvious that this is only used once.

Reply:
ServerVerifyInput is essentially a collection of references, so a reference to it is sufficient. Seeing this as an owned type makes me think it might contain owned memory, might be useful to pass off somewhere else or transform, etc.

I don't understand the comment about making it obvious that `ServerVerifyInput` is used only once.